packages/fetchai/connections/p2p_libp2p/dht/dhtpeer/dhtpeer.go

```diff
@@ -762,13 +891,21 @@ func (dhtPeer *DHTPeer) handleAeaNotifStream(stream network.Stream) {
 			}
 
 		}
+		duration := timer.GetTimer(start)
+		opLatencyRegister.Observe(float64(duration.Microseconds()))
 	}
 	dhtPeer.addressAnnounced = true
 }
 
 func (dhtPeer *DHTPeer) handleAeaRegisterStream(stream network.Stream) {
 	lerror, _, linfo, _ := dhtPeer.getLoggers()
 
+	nbrClients, _ := dhtPeer.monitor.GetCounter("service_relay_clients_count_all")
```

Comment:
This strings could be defined as constants elsewhere so we aid reuse and avoid magic values